posting/mvcc.go

```diff
@@ -150,15 +161,13 @@ func (ir *incrRollupi) Process(closer *z.Closer) {
 					delete(m, hash)
 				}
 			}
-		case <-forceRollupTick.C:
-			batch = ir.keysPool.Get().(*[][]byte)
-			if len(*batch) > 0 {
-				doRollup()
-			} else {
-				ir.keysPool.Put(batch)
-			}
-		case batch = <-ir.keysCh:
-			doRollup()
+		case batch := <-ir.priorityKeys[0].keysCh:
+			doRollup(batch, 0)
+			// Probably we don't need a limiter here because we expect not to call this function too frequently.
+
+		case batch := <-ir.priorityKeys[1].keysCh:
+			doRollup(batch, 1)
+
 			// throttle to 1 batch = 64 rollups per 100 ms.
```

Comment:
Is `1ms` sufficient throttle? Is so, fix this comment.

Reply:
Fixed the comment.